dynamic_programming/edit_distance.cpp

```diff
@@ -75,8 +75,10 @@ int editDistDP(string str1, string str2, int m, int n) {
 }
 
 int main() {
-    string str1 = "sunday";
-    string str2 = "saturday";
+    string str1;
+    string str2;
+
+    cin >> str1 >> str2;
```

Comment:
Can we have interactive string inputs:

`
cout << "Enter first string: ";
cin >> str1 ;

cout << "Enter second string: ";
cin >> str2 ;`